payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json

```diff
@@ -0,0 +1,1728 @@
+							},
+							"response": []
+						},
+						{
+							"name": "5b. Create Transfer for third party payments",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"var jsonData = JSON.parse(responseBody);",
+											"if (jsonData.id) postman.setEnvironmentVariable(\"new-transfer-id\", jsonData.id);"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"targetAccount\": {{new-recipient-id}},\n    \"quote\": \"{{new-quote-id}}\",\n    \"originalTransactionId\": \"<idempotency guid>\",\n    \"details\": {\n        \"reference\": \"Holiday\"\n    },\n    \"originator\": {\n        \"legalEntityType\": \"PRIVATE\",\n        \"reference\" : \"<unique customer id in your system>\",\n        \"name\": {\n            \"givenName\": \"John\",\n            \"middleNames\": [\n                \"Jay\"\n            ],\n            \"familyName\": \"Doe\"\n        },\n        \"dateOfBirth\": \"1977-07-01\",\n        \"address\": {\n            \"firstLine\": \"53 Shoreditch \",\n            \"city\": \"Tallinn\",\n            \"countryCode\": \"EE\",\n            \"postCode\": \"12112\"\n        }\n    }\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{host}}/v1/profiles/{{personal-profile-id}}/third-party-transfers",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"v1",
+										"profiles",
+										"{{personal-profile-id}}",
+										"third-party-transfers"
+									]
+								}
+							},
+							"response": []
+						}
+					],
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "3. Fund transfer",
+			"item": [
+				{
+					"name": "3a. Get payment reference for domestic transfer",
+					"item": [
+						{
+							"name": "1. Get payment reference (for bank payment funding)",
+							"protocolProfileBehavior": {
+								"disableBodyPruning": true
+							},
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									}
+								],
+								"body": {
+									"mode": "formdata",
+									"formdata": []
+								},
+								"url": {
+									"raw": "{{host}}/v1/quotes/{{new-quote-id}}/pay-in-methods",
```

Comment:
Should this definitely be v1 even for a v2 quote? quite confusing 😞

Reply:
@JamesFerretti , for this, there is no v2.

Reply:
Col